autogpt/llm/providers/openai.py

```diff
@@ -327,11 +328,13 @@ def __dict__(self):
         }
 
 
-def get_openai_command_specs(agent: Agent) -> list[OpenAIFunctionSpec]:
+def get_openai_command_specs(
+    command_registry: CommandRegistry, config: Config
```

Comment:
Just pass in `config.openai_functions` instead of the whole config.

Reply:
I'll remove that argument altogether.